examples/lsm-nice/lsm-nice-ebpf/src/main.rs

```diff
@@ -27,22 +28,22 @@ pub fn task_setnice(ctx: LsmContext) -> i32 {
 
 // (3)
 unsafe fn try_task_setnice(ctx: LsmContext) -> Result<i32, i32> {
+
     let p: *const task_struct = ctx.arg(0);
     let nice: c_int = ctx.arg(1);
     let ret: c_int = ctx.arg(2);
 
-    // If previous eBPF LSM program didn't allow the action, return the
-    // previous error code.
+    info!(&ctx, "The PID supplied to this program is: {}, with nice value {} and return value {}", (*p).pid, nice, ret); 
+    info!(&ctx, "The PID that we need to monitor for changes: {}", core::ptr::read_volatile(&PID));
```

Comment:
should this say "...for changes is:"? the word "is" is used on the previous log, but not this one.

you might also consider merging the two log lines. logs aren't free :)

lastly: you're now doing two `core::ptr::read_volatile`s. can you do just one?